src/macad_gym/viz/render.py

```diff
@@ -25,8 +27,9 @@ def multi_view_render(images, unit_dimension, actor_configs):
     for actor_id, im in images.items():
         if not actor_configs[actor_id]["render"]:
             continue
-        surface = pygame.surfarray.make_surface(im.swapaxes(0, 1) * 128 + 128)
-        surface_seq += ((surface, (poses[actor_id][1], poses[actor_id][0])), )
+        im = cv2.resize(im, (unit_dimension[1], unit_dimension[0]))
```

Comment:
I believe `im` in `images` are already in the correct size per the env config's `x_res` and `y_res`. Is this `resize(...)` necessary?